examples/conanfile/package_id/settings_and_options.rst

```diff
@@ -0,0 +1,145 @@
+.. _examples_conanfile_package_id_settings_options:
+
+How settings and options of a recipe influence its pacakge ID
```

Comment:
The ``conan list`` command can be very useful to illustrate this section, using it after the ``conan create`` it is possible to see the different binaries with different settings, not only hte "package-id" which in isolation is less explicit

Reply:
Done